docs/known_issues.rst

```diff
@@ -16,3 +16,8 @@ Known issues
   quotes in the yaml file. This is due to the way the yaml file is parsed: with
   double quotes, special characters (like ``\t``) will be expanded, which can
   lead to crashes.
+
++ Special care should be taken when using the backslash character (``\``),
```

Comment:
Special care should be taken when using the backslash character (``\``) *in regexes*